examples/wines.get.json

```diff
@@ -708,11 +730,11 @@
                         },
                         {
                             "locale": "fr",
-                            "value": "<ul><li>​Allgemeiner <i>Hinweis</i></li></ul>"
+                            "value": "<ul><li>Allgemeiner <i>Hinweis</i></li></ul>"
```

Comment:
Removed a zero-width space character between "<li>" and "Allgemeiner" .. same for the coming changes